mne/time_frequency/tfr.py

```diff
@@ -302,12 +306,15 @@ def _make_dpss(
                 real_offset = Wk.mean()
                 Wk -= real_offset
             Wk /= np.sqrt(0.5) * np.linalg.norm(Wk.ravel())
+            Ck = np.sqrt(conc[m])
 
             Wm.append(Wk)
+            Cm.append(Ck)
 
         Ws.append(Wm)
+        Cs.append(Cm)
     if return_weights:
-        return Ws, conc
+        return Ws, Cs
```

Comment:
The existing approach was to just return `conc` as the weights, however I believe this is incorrect, as `conc` refers only to those weights from the last frequency looped through. The `Cs` I am using now is for all tapers **and** all frequencies.

Reply:
oops, yeah, makes sense.